events/event/forms.py

```diff
@@ -1,17 +1,18 @@
 from django import forms
 from .models import Event
 from datetime import datetime
+from pytz import utc as TZ
 
 class DateValidator(forms.Field):
     def validate(self, value):
         time_now = datetime.now().timestamp()
         if not value:
             raise forms.ValidationError("This field is required")
         try:
-            float(value)
+            time = float(value)
```

Comment:
This will work but I do not recommend to call variables like this. "time" is name of module from standard library. variable with the same name can cause some namespace issues.